docs/home/howto/UPGRADE.md

```diff
@@ -457,3 +457,166 @@ directories by running `delete_old_cluster.sh` script (created in `pg_upgrade_wo
 nodes**. The script is not created if you have user-defined tablespaces inside the old data directory. You can also
 delete the old installation directories (e.g., `bin`, `share`). You may delete `pg_upgrade_working_dir`
 on primary node once the upgrade is completely over.
+
+## Terraform upgrade from Epiphany 1.x to 2.x
+
+From Epiphany 1.x to 2.x the Terraform stack received the following major updates:
+- Terraform 0.12.6 to 1.1.3
+- Azurerm provider 1.38.0 to 2.91.0
+- AWS provider 2.26 to 3.71.0
+
+These introduce some breaking changes which will require manual steps for upgrading an existing 1.x clusters. As this is not straight forward we recommand deploying a new cluster on 2.x and migrating data instead.
+
+If you do want to upgrade a 1.x manually it will require you to upgrade between a few different versions of Terraform. The basic steps are described below for each provider. As always insure backup of any data required.
+
+Final note is that you can also leave the Terraform in place and use the `--no-infra` flag to skip applying the new Terraform scripts. This however makes you unable to make any changes to your cluster layout.
+
+### Azure
+
+Notes:
+- If you made any manual changes to your cluster infrastructure outside of Terraform this might cause issues.
+- Some resources might be changed or added which are usually security-groups and security-groups-association or NIC's. This is normal behaviour and is ok, just make sure no other resources like VM's are included when reviewing the `terraform plan` output.
+- Only run `terraform apply` if `terraform plan` shows your infrastructure does not match the configuration.
+
+#### v0.12.6 => v0.13.x
+
+Offical documentation can be found here: https://www.terraform.io/language/upgrade-guides/0-13
```

Comment:
```suggestion
Official documentation can be found here: https://www.terraform.io/language/upgrade-guides/0-13
```

Reply:
And it occurs in more lines below.

Reply:
There are occurrences of `Offical` still in lines: 461. 476, 491, 527, 541, 556, 570 in the same file. These lines should be corrected as well.

Reply:
All fixed.